projects/storefrontstyles/scss/components/configurator/_config-attribute-header.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../theme';
+
+%cx-config-attribute-header {
+  .cx-config-attribute-label {
+    padding-top: 10px;
+    padding-right: 6px;
+    font-size: $font-size-base;
+    font-weight: bold;
+    display: block;
+    clear: left;
```

Comment:
Do we still need `clear`s in styles files?

Reply:
done.